.github/workflows/deploy.yml

```diff
@@ -0,0 +1,86 @@
+
+on:
+  workflow_call:
+    inputs:
+      workspace_name:
+        description: 'The terraform workspace to target'
+        required: true
+        type: string
+      version_tag:
+        description: 'Container version to deploy'
+        required: true
+        type: string
+    outputs:
+      cluster_config:
```

Comment:
Just double check this doesn't have anything too sensitive in it like secrets. Doesn't look like it